fastos/src/vespa/fastos/linux_file.cpp

```diff
@@ -37,7 +37,18 @@ FastOS_Linux_File::FastOS_Linux_File(const char *filename)
 ssize_t
 FastOS_Linux_File::readInternal(int fh, void *buffer, size_t length, int64_t readOffset)
 {
-    return File_RW_Ops::pread(fh, buffer, length, readOffset);
+    char * data = static_cast<char *>(buffer);
+    ssize_t read(0);
```

Comment:
Consider a different name for the variable, e.g. `has_read`

Reply:
done